clients/algoliasearch-client-python/algoliasearch/http/hosts.py

```diff
@@ -6,9 +6,14 @@ class Host:
     TTL = 300.0
 
     def __init__(
-        self, url: str, priority: Optional[int] = 0, accept: Optional[int] = None
+        self,
+        url: str,
+        scheme="https",
```

Comment:
```suggestion
        scheme: str ="https",
```

not sure if type inference works here